sdk/core/azure-core/src/main/java/com/azure/core/implementation/http/rest/SyncRestProxy.java

```diff
@@ -80,22 +80,17 @@ public Object invoke(Object proxy, Method method, RequestOptions options, EnumSe
 
             return handleRestReturnType(decodedResponse, methodParser, methodParser.getReturnType(), context, options,
                 errorOptions);
-        } catch (RuntimeException e) {
+        } catch (Exception e) {
             tracer.end(null, e, context);
-            throw LOGGER.logExceptionAsError(e);
-        } finally {
-            try {
-                scope.close();
-            } catch (Exception e) {
-                LOGGER.verbose("Failed to close scope");
-            }
+            sneakyThrows(e);
```

Comment:
:ok_hand:

Reply:
Reading through some of OpenTelemetry's code in Tracers I'm not even sure this will be reached. 😄